### PR TITLE
Added contribution section to readme

### PR DESCRIPTION
**Pull request by glimchb**
from https://github.com/opiproject/opi-poc/pull/4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,9 @@ OPI subgroup dealing with provisioning and lifecycle management.
     * [lifecycle-subgroup](https://opi-project.slack.com/archives/C0342L6T7EC)
 * Status
     * Active
+
+# I Want To Contribute!
+
+This project welcomes contributions and suggestions.  We are happy to have the Community involved via submission of **Issues and Pull Requests** (with substantive content or even just fixes). We are hoping for the documents, test framework, etc. to become a community process with active engagement.  PRs can be reviewed by by any number of people, and a maintainer may accept.
+
+See [CONTRIBUTING](../opi/CONTRIBUTING.md) and [GitHub Basic Process](../opi/doc-github-rules.md) for more details.
```
Comment (mestery) [inline README.md:25]: Does this link work across repos?

Comment (glimchb) [reply README.md:25]: nope

Comment (mestery) [reply README.md:25]: OK, you would need to wait until https://github.com/opiproject/opi/pull/3 merges and then put the full link in.

Comment (glimchb) [reply README.md:25]: yep

**Review by mestery — APPROVED** [review]
image

**Pull request merged**